libs/SalesforceSDK/build.gradle

```diff
@@ -2,6 +2,8 @@
 apply plugin: 'android-library'
 
 dependencies {
+  compile 'com.android.support:appcompat-v7:23.0.0'
+  compile 'com.facebook.react:react-native:0.11.+'
```

Comment:
I will leave the class and address the fixme when I figure out how to do the template app for react native.